packages/os-installer-commit/os-installer-commit.pacscript

```diff
@@ -0,0 +1,36 @@
+name="os-installer-commit"
+maintainer="cat-master21 <96554164+cat-master21@users.noreply.github.com>"
+pkgname="os-installer"
+description="A generic and changable os-installer (latest commit)"
+url="https://gitlab.gnome.org/p3732/os-installer/-/archive/${version}/os-installer-${version}.tar.gz"
+depends="libgnome-desktop-4-2 libadwaita-1-0 libgweather-4-0 python3-yaml libudisks2-0 libvte-2.91-gtk4-0"
+build_depends="meson gettext libgnome-desktop-4-dev libadwaita-1-dev libgweather-4-dev python3-yaml libudisks2-dev libvte-2.91-gtk4-dev"
+breaks="${pkgname}-bin ${pkgname} ${pkgname}-git"
+optdepends=("gnome-disk-utility: Default disk CMD if not set"
+            "epiphany-browser: Default browser CMD if not set"
+            "gnome-control-center: Default wifi CMD if not set")
+
+pkgver() {
+  wget -q -O - https://gitlab.gnome.org/p3732/os-installer/-/commits/main | grep 'data-clipboard-text="' | head -1 | cut -f14 -d'"'
```

Comment:
Can't you just use `git ls-remote`?

Reply:
no becuase of the issue with the RL ISO builder